src/CommonLib/Processors/LdapPropertyProcessor.cs

```diff
@@ -520,8 +520,16 @@ public class LdapPropertyProcessor {
         /// <param name="entry"></param>
         public Dictionary<string, object> ParseAllProperties(IDirectoryObject entry) {
             var props = new Dictionary<string, object>();
+
+            var type = typeof(LDAPProperties);
```

Comment:
This is going to revert a change we made in the newest commits. Instead, you'll need to add dsasignature to the ReservedAttributes list above and then use that.